tapcfg/server.go

```diff
@@ -373,6 +373,7 @@ func genServerConfig(cfg *Config, cfgLogger btclog.Logger,
 			ChannelLister:   walletAnchor,
 			AliasManager:    lndRouterClient,
 			// nolint: lll
+			AcceptPriceDeviationPpm:   rfqCfg.AcceptPriceDeviationPpm,
```

Comment:
If a `//nolint` comment is above a line, it only counts for that line. But both of these are too long. So either needs a comment each, or put the comment before a block, so it counts for the whole block (so for example before `rfq.ManagerCfg{` on line 369).